src/components/ChatLog.js

```diff
@@ -3,25 +3,25 @@ import './ChatLog.css';
 import ChatEntry from './ChatEntry';
 import PropTypes from 'prop-types';
 
-const ChatLog = (props) => {
-  const chatComponents = props.entries.map((entry, index) => {
+const ChatLog = ({ entries, onLikeClicked }) => {
+  const chatComponents = entries.map((entry, index) => {
     return (
-      <p key={index}>
+      <div key={index}>
         <ChatEntry
           id={entry.id}
           sender={entry.sender}
           body={entry.body}
           timeStamp={entry.timeStamp}
           liked={entry.liked}
-          onUpdate={props.onUpdateEntry}
+          onLikeClicked={() => onLikeClicked(entry.id)}
         ></ChatEntry>
-      </p>
+      </div>
     );
   });
 
   return (
     <section>
-      <p>{chatComponents}</p>
+      <div>{chatComponents}</div>
```

Comment:
I just changed these to div because my dev console was complaining about <p> and <h2> (from ChatEntry) being nested in a <p> (ChatLog)